tests/pkg_fatfs_vfs/Makefile

```diff
@@ -25,14 +25,15 @@ else
   USEMODULE += mtd_sdcard
 endif
 
-BOARD_INSUFFICIENT_MEMORY := nucleo-f031k6
+BOARD_INSUFFICIENT_MEMORY := msb-430 msb-430h nucleo-f031k6 telosb \
```

Comment:
I also have issue with my ubuntu stable for building these, I will try to find why.